tracer/test/Datadog.Trace.Tests/Propagators/MultiSpanContextPropagatorTests.cs

```diff
@@ -36,11 +43,11 @@ public void Inject_All_IHeadersCollection()
             headers.Verify(h => h.Set(HttpHeaderNames.TraceId, "123456789"), Times.Once());
             headers.Verify(h => h.Set(HttpHeaderNames.ParentId, "987654321"), Times.Once());
             headers.Verify(h => h.Set(HttpHeaderNames.SamplingPriority, "2"), Times.Once());
-            headers.Verify(h => h.Set(B3ContextPropagator.TraceId, "00000000075bcd15"), Times.Once());
-            headers.Verify(h => h.Set(B3ContextPropagator.SpanId, "000000003ade68b1"), Times.Once());
-            headers.Verify(h => h.Set(B3ContextPropagator.Sampled, "1"), Times.Once());
+            headers.Verify(h => h.Set(B3MultipleHeaderContextPropagator.TraceId, "00000000075bcd15"), Times.Once());
+            headers.Verify(h => h.Set(B3MultipleHeaderContextPropagator.SpanId, "000000003ade68b1"), Times.Once());
+            headers.Verify(h => h.Set(B3MultipleHeaderContextPropagator.Sampled, "1"), Times.Once());
```

Comment:
nit: I wonder if we should use the raw expected string values in all these tests, given they're standard values, to check for typos in the constants?

Reply:
Done.